autoload/airline/extensions.vim

```diff
@@ -444,6 +444,12 @@ function! airline#extensions#load()
       endif
     endfor
   endif
+
+  if exists(':Dirvish')
```

Comment:
You should add this check

```viml
if exists(':Dirvish') && get(g:, 'airline#extensions#dirvish#enabled', 1)
```

Reply:
Will add this in the next commit I make.